ui/pages/keyring-snaps/snap-account-detail-page/snap-account-detail-page.test.tsx

```diff
@@ -162,6 +162,23 @@ describe('SnapAccountDetails', () => {
   it('it should render update if snap update is available', async () => {
     const mockStateForConfig = {
       metamask: {
+        snapRegistryList: {
+          'a51ea3a8-f1b0-4613-9440-b80e2236713b': {
+            id: 'a51ea3a8-f1b0-4613-9440-b80e2236713b',
+            snapId: 'npm:@metamask/snap-simple-keyring',
+            iconUrl: '',
+            snapTitle: 'Metamask Simple Keyring',
```

Comment:
```suggestion
            snapTitle: 'MetaMask Simple Keyring',
```

Reply:
These are mock data, i'll make the appropriate changes to the registry